src/dune_rules/dune_loaded_file.mli

```diff
@@ -0,0 +1,21 @@
+(* Separate from [Dune_load] to break dependency cycles *)
+
+open! Dune_engine
+open! Stdune
+
+(** A fully evaluated dune file *)
+type t =
+  { dir : Path.Source.t
+  ; project : Dune_project.t
+  ; stanzas : Dune_file.Stanzas.t
+  }
```

Comment:
You could also move this to `Dune_file`, so the the type can be `Dune_file.t` I would then prefer to start splitting away other stanza components from `Dune_file`.

Reply:
What would we name the existing `Dune_file` module in dune_rules? I'm not opposed to this change in principle, but I think I'm misunderstanding what you're asking for

Reply:
@cwong-ocaml, I think Rudi proposes to move the record definition into the existing module, which seems reasonable to me.
Note that Dune_file.t currently does not exist, so it looks like destiny that the name was kept available. :-)

Reply:
Yep, this is what I meant. Thanks for clarifying.

Reply:
Done! That seems like a much better idea.